backend/lcfs/web/api/fuel_supply/repo.py

```diff
@@ -220,7 +222,7 @@ async def get_fuel_supply_list(self, compliance_report_id: int) -> List[FuelSupp
 
     @repo_handler
     async def get_fuel_supplies_paginated(
-        self, pagination: PaginationRequestSchema, compliance_report_id: int
+        self, pagination: PaginationRequestSchema, compliance_report_id: int = None, effective: bool = True
```

Comment:
Looks like this function needs the compliance_report_id, so lets remove the default as it should always be passed in.